modules/ksql-cluster/ksql-cluster.tf

```diff
@@ -0,0 +1,13 @@
+resource "confluent_ksql_cluster" "ksqldb_cluster" {
+  display_name = "example"
```

Comment:
are we going to use `example` as the display name? What does that mean? Is it the name of the cluster? Should this be parameterized?

Reply:
fixed